2011-10-14 14:26:53,750 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,774 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,775 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,790 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,792 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,806 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,807 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,822 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,823 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,838 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,839 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,855 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCLattice', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,856 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,885 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_Ar_P_MLJ_MI_OPBC_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,886 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,908 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,910 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,926 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,927 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,944 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,945 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,961 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,962 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,979 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,980 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:53,996 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCLattice', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:53,997 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.BCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCLattice.py", line 30, in BCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:26:54,023 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:26:54,024 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Al_PF_ErcolessiAdams', element='Al', info:PotentialLoadFailed()
2011-10-14 14:26:54,025 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:26:55,301 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:26:55,301 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ne_P_LJ_NEIGH_PURE_H', element='Ne', info:PotentialLoadFailed()
2011-10-14 14:26:55,302 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:26:55,310 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:26:55,310 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_C', element='Ar', info:PotentialLoadFailed()
2011-10-14 14:26:55,311 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:26:55,319 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:26:55,319 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ne_P_MLJ_NEIGH_RVEC_F', element='Ne', info:PotentialLoadFailed()
2011-10-14 14:26:55,320 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:26:55,328 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:26:55,328 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_F90', element='Ar', info:PotentialLoadFailed()
2011-10-14 14:26:55,329 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:26:55,341 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_MI_OPBC_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:26:55,342 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,050 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,051 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,062 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,063 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,078 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,079 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,095 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,097 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,114 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,115 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,132 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,133 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 135, in results
    positions = self.makePerturbedLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 47, in makePerturbedLattice
    a = self.getFCCLattice()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 23, in getFCCLattice
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:27:59,146 - openkimtests.potential.KIM_loader - WARNING - KIM Potential needs slab information
2011-10-14 14:27:59,147 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='ForceTest', potential='model_Ar_P_MMorse', element='Ar', info:PotentialLoadFailed()
2011-10-14 14:27:59,148 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 137, in results
    gradE = self.gradE(positions)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 91, in gradE
    pe_x1 = self.getPotential(atoms_list, positions+perturb)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/ForceTest.py", line 63, in getPotential
    atoms.set_calculator(self.calculator)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 59, in calculator
    self.slab)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 238, in load
    return potentials[name](name,element,slab,*args,**kwargs)
  File "/a/yjchen/openkimtest/openkimtests/bin/potential.py", line 115, in KIM_loader
    raise PotentialLoadFailed('KIM',*args,**kwargs)
PotentialLoadFailed: KIM)

2011-10-14 14:27:59,185 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Al_PF_ErcolessiAdams', element='Al', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:27:59,188 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,619 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ne_P_LJ_NEIGH_PURE_H', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,620 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,635 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,636 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,654 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ne_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,656 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,673 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,675 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,694 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_MI_OPBC_H_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,695 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,721 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,722 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,739 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,740 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,757 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,758 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,775 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,776 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,793 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,794 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,812 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,813 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,832 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='BCCElasticConstants', potential='model_Ar_P_MMorse', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,834 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/BCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,858 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Al_PF_ErcolessiAdams', element='Al', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,859 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,882 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ne_P_LJ_NEIGH_PURE_H', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,883 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,897 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,899 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,918 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ne_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,919 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,938 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:35,940 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:35,958 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_MI_OPBC_H_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,959 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,982 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,982 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:35,996 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:35,996 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:36,013 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:36,014 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:36,031 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:36,031 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:36,051 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:36,052 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:36,072 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:UnboundLocalError("local variable 'testObject' referenced before assignment",)
2011-10-14 14:28:36,073 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 56, in run_test
    del testObject
UnboundLocalError: local variable 'testObject' referenced before assignment

2011-10-14 14:28:36,095 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCElasticConstants', potential='model_Ar_P_MMorse', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:36,097 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 63, in results
    elastic_constants = hess(sp.zeros(6))
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 1202, in __call__
    return self._hessian(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 819, in _hessian
    hess = self._hessdiag(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 807, in _hessdiag
    return self._partial_der(x00)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 780, in _partial_der
    PD[ind] = self._derivative(x0[ind], stepNom[ind])
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 708, in _derivative
    f_x0[j] = self.fun(x0j)
  File "/usr/local/pub/lib/python2.7/site-packages/Numdifftools-0.3.5-py2.7.egg/numdifftools/core.py", line 792, in _fun
    return  self.fun_org(x)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCElasticConstants.py", line 54, in strain
    energy = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:36,476 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Al_PF_ErcolessiAdams', element='Al', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:36,477 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:38,518 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ne_P_LJ_NEIGH_PURE_H', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:38,519 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:38,901 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:38,902 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:39,196 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ne_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:39,197 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:39,634 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:39,636 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:28:39,655 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_MI_OPBC_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:28:39,656 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,732 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_CLUSTER_F90', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,733 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,748 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,749 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,765 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,766 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,786 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_ArNe_P_MLJ_NEIGH_RVEC_F', element='Ne', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,787 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,807 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_CLUSTER_C', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,808 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:51,829 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MLJ_NEIGH_PURE_H_F', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:51,830 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 16, in results
    a = request(self.potentialname,self.element,'FCCLattice','FCCLatticeConstant')
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 150, in request
    run_test(test,potential,element)
  File "/a/yjchen/openkimtest/openkimtests/bin/pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 43, in results
    minimum, energyminimum, iterations, funcalls, warnflag = fmin(self.FCCEnergy,x0,full_output=1,disp=0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 278, in fmin
    fsim[0] = func(x0)
  File "/usr/local/pub/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 176, in function_wrapper
    return function(x, *args)
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCLattice.py", line 30, in FCCEnergy
    return self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

2011-10-14 14:29:52,218 - openkimtests.pipeline.run_tests - ERROR - 
ERROR on test='FCCVacancy', potential='model_Ar_P_MMorse', element='Ar', info:RuntimeError('LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.',)
2011-10-14 14:29:52,220 - openkimtests.pipeline.run_tests - ERROR - TRACEBACK: Traceback (most recent call last):
  File "pipeline.py", line 126, in run_tests
    verbose=verbose)
  File "pipeline.py", line 51, in run_test
    results = testObject.main()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/_BaseTest.py", line 83, in main
    results = self.results()
  File "/a/yjchen/openkimtest/openkimtests/kim_tests/FCCVacancy.py", line 24, in results
    energy1 = self.slab.get_potential_energy()
  File "/a/yjchen/ase/ase/atoms.py", line 526, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 118, in get_potential_energy
    self.update(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 134, in update
    self.calculate(atoms)
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 138, in calculate
    self.run()
  File "/a/yjchen/openkimtest/openkimtests/bin/kim.py", line 241, in run
    (cwd,exitcode))
RuntimeError: LAMMPS exited in /a/yjchen/openkimtest/openkimtests/tmp with exit code: 71.

